src/main/resources/templates/fragments.html

```diff
@@ -16,31 +16,48 @@
 
 <body>
 
-    <nav th:fragment="page-header" class="navbar navbar-expand-md bg-success navbar-dark navbar-fixed-top">
```

Comment:
Major navbar updates for button to work then to keep the styling similar to what we had previously.